apps/molgenis-components/src/components/forms/InputRadio.vue

```diff
@@ -68,11 +73,16 @@ export default {
   },
   computed: {
     isClearShown() {
-      return (
-        this.isMouseOver &&
-        this.radioValue !== null &&
-        this.radioValue != undefined
-      );
+      if(this.isClearable !== undefined && this.isClearable === false) {
```

Comment:
I think isClearable is a good option. However, we might need in the future to go a step further and consider to change 'boolean' to actually have 'yes', 'no', 'unknown' states to be explicit on the meaning of 'clear'